CONTRIBUTING.md

```diff
@@ -1,8 +1,11 @@
 # CONTRIBUTING
 
-The libuv project welcomes new contributors. This document will guide you
+The "Port of libuv to IBM® z Systems" project welcomes new contributors. This document will guide you
```

Comment:
@Paul-Ivinson CLA.md also contains the z Systems reference.  Where does that exactly get tied in?   Can we not generalize that as well?  "libuv to IBM® z Systems"